Lib/fontTools/ttLib/tables/_n_a_m_e.py

```diff
@@ -120,6 +120,37 @@ def getDebugName(self, nameID):
 		else:
 			return None
 
+	def getFamilyName(self):
+		for nameID in (21, 16, 1):
+			name = self.getDebugName(nameID)
+			if name is not None:
+				return name
+		return None
+
+	def getSubFamilyName(self):
+		for nameID in (22, 17, 2):
+			name = self.getDebugName(nameID)
+			if name is not None:
+				return name
+		return None
```

Comment:
you do basically the same thing in getFamilyName and getSubFamilyName, with different list of ids, maybe you can factor out this logic into a method named `getBestDebugName` which takes a sequence of name ids integers, and returns one if it finds it or None, so you can use in both of these.